### DISCUSSION TITLE
DevPool Business Development

### DISCUSSION THREAD

**Discussion by sergfeldman** [ubiquity/.github#12] [DevPool]
DevPool Business Development strategies 

### 1. Invite top projects 
- Contact top Web3 or DeFI projects on GitHub (Uniswap, etc)
- Contacts Ubiquity investors in DMs (Animoca Brands, etc)
- Contact top VC via their website/discord (Paradigm, etc)

### 2. Invite projects with open bounties
- Scrap open bounties from other places (Gitcoin, etc)

**Comment by sergfeldman** [comment]
Top Web3 or DeFI projects

1
https://ethereum.org/
https://makerdao.com/
https://uniswap.org/
https://consensys.net/ https://metamask.io/
https://opensea.io/

2
https://thegraph.com/
https://chain.link/
https://lido.fi/
https://aave.com/
https://polygon.technology/

**Reply by 0x4007** [reply]
Seems a bit limited of a list. I expected 20+ 

I suggest you also look at https://sovs.notion.site/sovs/Sov-s-Compendium-41f097d28dae4d09801f10cde1b2d03b

Sov's compendium 

Sov keeps a really good pulse on intelligence in the crypto community. I'm sure he has a list somewhere that answers this for us.

**Reply by sergfeldman** [reply]
> Seems a bit limited of a list. I expected 20+
> 
> I suggest you also look at https://sovs.notion.site/sovs/Sov-s-Compendium-41f097d28dae4d09801f10cde1b2d03b
> 
> Sov's compendium
> 
> Sov keeps a really good pulse on intelligence in the crypto community. I'm sure he has a list somewhere that answers this for us.

@pavlovcik The intention was to compile top 5 and top 10 lists just to start.
Thank you, will analyze Sov's compedium and extend the list to 20+.

**Reply by sergfeldman** [reply]
https://compound.finance/
https://synthetix.io/
https://curve.fi/
https://pancakeswap.finance/
TBD

**Comment by sergfeldman** [comment]
Top VC 

1
https://a16zcrypto.com/
https://labs.binance.com/ 
https://www.paradigm.xyz/
https://www.coinbase.com/ventures
https://www.animocabrands.com/

2
https://www.sequoiacap.com/
https://www.blocklabs.io/funding 
https://polychain.capital/
https://panteracapital.com/
https://jumpcrypto.com/

**Reply by 0x4007** [reply]
Also refer to Sov list

**Reply by sergfeldman** [reply]
https://blockchain.capital/
https://www.dragonfly.xyz/ 
https://au21.capital/
https://genblock.capital/
https://www.signum.capital/

Large but problematic
https://shima.capital/
https://dcg.co/

**Comment by sergfeldman** [comment]
Post to the GitHub issue comments:

Hey, we are the decentralized DevPool  (link https://dao.ubq.fi/devpool)

The DevPool helps to scale Web3 development while paying only for completed issues - bounties.

We propose to resolve this GitHub issue for free, if there is a possibility of a longterm partnership on bounties.

Who makes such decisions for (title of the project)?

**Comment by 0x4007** [comment]
Some R&D teams I would love to build a relationship with because we use their products:

- Gnosis (Safe, CowSwap)
- Uniswap (Uniswap V3)
- Curve (MetaPool)
- Paradigm (Foundry)

Others with top notch teams:

- MakerDAO
- Synthetix 
- Optimism
- Liquity

**Reply by 0x4007** [reply]
Some traditional companies that would be interesting to build a relationship with

- Microsoft https://github.com/microsoft/TypeScript/issues
- GitHub (Grants to incentivize open source development)

---

For research: https://github.com/collections/open-source-organizations

**Comment by 0x4007** [comment]
DevPool CRM Draft

https://docs.google.com/spreadsheets/d/1EUoudawB3Dc_qUtOKajRomBBRBWyXl6K6d7FyAAqZ-w/edit#gid=0

**Comment by 0x4007** [comment]
https://twitter.com/imperiumpaper/status/1621853040830734336?s=46&t=aJrgKWT4Yync1ROjKuT4Nw

**Reply by 0x4007** [reply]
On chain governance DAOs:
- [@Uniswap](https://twitter.com/Uniswap)
- [@MakerDAO](https://twitter.com/MakerDAO)
- [@AaveAave](https://twitter.com/AaveAave)
- [@compoundfinance](https://twitter.com/compoundfinance)
- [@LidoFinance](https://twitter.com/LidoFinance) 
- [@CurveFinance](https://twitter.com/CurveFinance)
